torch/_torch_docs.py

```diff
@@ -4926,6 +4926,62 @@
 
 """)
 
+add_docstr(torch.unique,
+           r"""
+unique(input, sorted=False, return_inverse=False) -> (Tensor, Tensor)
+
+Returns the unique scalar elements of the input tensor as list.
+
+Args:
+    input (Tensor): the input tensor
+    sorted (bool): Whether to sort the unique elements in ascending order
+        before returning as output.
+    return_inverse (bool): Whether to also return the indices for where
+        elements in the original input ended up in the returned unique list.
+
+Returns:
+    (Tensor, Tensor): A tuple containing
+
+        - **output** (*Tensor*): the list of unique scalar elements
+        - **inverse_indices** (*Tensor*): the indices (same shape as input)
+            for where elements in the original input map to in the output
+            if ``return_inverse`` is ``True``; otherwise, an empty tensor.
```

Comment:
Please format the docs according to the format we use everywhere else

Reply:
This is following the format from `torch.eig` - can you clarify which one I should follow instead? In particular, I was looking for a format that returns a tuple of Tensor and clearly documents what each of the returned tensor means. The `Returns:` section seems to be inconsistently used - is that not part of the format?